src/vulkan/vulkan_presenter.cpp

```diff
@@ -105,8 +105,20 @@ namespace dxvk::vk {
     VkResult status;
     
     if ((status = m_vki->vkGetPhysicalDeviceSurfaceCapabilitiesKHR(
-        m_device.adapter, m_surface, &caps)) != VK_SUCCESS)
-      return status;
+        m_device.adapter, m_surface, &caps)) != VK_SUCCESS) {
+      for (uint32_t i = 0; i < 5 && status == VK_ERROR_SURFACE_LOST_KHR; i++) {
```

Comment:
Is there a valid scenario where this can fail an arbitrary number of times in a row? I'd prefer a solution with no more than *one* attempt if that is reasonable, since any functions that affect the window should have completed at this point.